src/ert/shared/models/multiple_data_assimilation.py

```diff
@@ -66,23 +66,23 @@ def runSimulations(
         update_id = None
         enumerated_weights = list(enumerate(weights))
         starting_iteration = self._simulation_arguments["start_iteration"]
-        restart_run = starting_iteration != 0
+        restart_run = self._simulation_arguments["restart_run"]
         case_format = self._simulation_arguments["target_case"]
+        prior_ensemble = self._simulation_arguments["prior_ensemble"]
         storage_manager = self.ert().storage_manager
         if restart_run:
-            if case_format % starting_iteration not in storage_manager:
+            assert starting_iteration > 0
```

Comment:
There is no test that runs this with `restart_run=True` it would be great if we could increase the coverage of this now that it is being changed.

Reply:
Is there an issue created for this already?